java/aquality-selenium-template/src/test/java/aquality/selenium/template/glue/stepdefinitions/DemoSteps.java

```diff
@@ -0,0 +1,61 @@
+package aquality.selenium.template.glue.stepdefinitions;
+
+import aquality.selenium.browser.AqualityServices;
+import aquality.selenium.template.configuration.Configuration;
+import aquality.selenium.template.forms.TopBarMenu;
+import aquality.selenium.template.forms.pages.ContactUsPage;
+import aquality.selenium.template.forms.pages.MainPage;
+import aquality.selenium.template.models.ContactUsInfo;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+
+import static aquality.selenium.template.forms.TopBarMenu.Item.CONTACT_US;
+
+public class DemoSteps {
+
+    private final MainPage mainPage = new MainPage();
+    private final TopBarMenu topBarMenu = new TopBarMenu();
+    private final ContactUsPage contactUsPage = new ContactUsPage();
+
+    @Given("Main page is opened")
+    public void checkMainPageIsOpened() {
```

Comment:
why is this method named "check"? Should we check anything inside? Should we?

Reply:
fixed